src/ScssFileNameGuesser.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Symfonycasts\SassBundle;
+
+class ScssFileNameGuesser
+{
+    public static function guessCssNameFromSassFile(string $sassFile, string $outputDirectory): string
+    {
+        $fileName = basename($sassFile, '.scss');
+
+        return $outputDirectory . '/' . $fileName . '.output.css';
```

Comment:
i think this should just be `$filename.css` so it aligns with the assetmapper docs nicely